src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -17,5 +17,5 @@
     public static final Prefix PREFIX_AND = new Prefix("/AND");
     public static final Prefix PREFIX_OR = new Prefix("/OR");
     public static final Prefix PREFIX_NOT = new Prefix("/NOT");
-    public static final Prefix PREFIX_COE_EXPIRY = new Prefix("coe/");
+    public static final Prefix PREFIX_COE_EXPIRY = new Prefix("ex/");
```

Comment:
Hmm why are you changign this, 
In the user guide its coe/

Find customers from the contact list that matches specified filters.

Format: `find  n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS c/OWNED_CARBRAND+OWNED_CARTYPE coe/COE_EXPIRY_DATE t/TAGs 
cp/PREFERRED_CARBRAND+PREFERRED_CARTYPE b/DATE_OF_BIRTH`

**IMP** : All arguments are optional. In particular:
- for `c/OWNED_CARBRAND+OWNED_CARTYPE`, user can either give brand or type information, or both using the `+`sign to 
  separate.
  
- for `cp/PREFERRED_CARBRAND+PREFERRED_CARTYPE` user can either give brand or type information, or both using the 
  `+` sign to separate.
  
- for `n/NAME` giving a partial name is also okay - all names which have the given parameter as a subsequence (Note 
  that `abc` is a subsequence of `addbddc`) will be matched. Note that this is the only field for which we match 
  a subsequence instead of a simple substring.
  
- for `e/EMAIL`, all email containing the given parameter will be valid, but note that domain name is not checked.
    - eg `find e/abc` will return profiles with the following emails:
        -  `abcd@gmail.com`
        -  `bbabc@gmail.com`
    - It will _not_ return:
        - `xxx@abc.com`
        - `axbyc@hhh.com`


Further details about the search options are as follows:

- For `coe/COE_EXPIRY_DATE`, using `coe/exp` will search for all customers with an expired COE on any of the cars they own.
  it is a special case alias for the search `coe/0`, where `coe/NON_NEGATIVE_NUMBER` will search for any customers with
  at least one car that will expire in `NON_NEGATIVE_NUMBER` years *or less*.